### PR TITLE
Feature/934 validators filters

### PR DESCRIPTION
**Pull request by matthewcarlreetz**

### CODE REVIEW

File: src/features/hotspots/root/HotspotsList.tsx [hunk 1]
```diff
@@ -123,22 +291,49 @@ const HotspotsList = ({
...
+            showCarot
+            loading={loadingHotspotRewards}
+            totalReward={hotspotRewards[item.address]}
+            hidden={hiddenAddresses?.includes(item.address)}
+          />
+        )
+      }
+      if (isValidator(item)) {
+        return (
+          <ElectedValidatorItem
```
Comment (tyler-whitman) [inline src/features/hotspots/root/HotspotsList.tsx:321]: Maybe we refactor this to `ValidatorListItem`?

Comment (matthewcarlreetz) [reply src/features/hotspots/root/HotspotsList.tsx:321]: that sounds good

**Review by tyler-whitman — APPROVED** [review]
Code looks good and working well for me, only other thing I noticed is that the HNT amount for the validator list item doesn't convert to local currency when using the toggle. Thats the way it works it is on the consensus list too though.

**Comment by matthewcarlreetz** [comment]
> Code looks good and working well for me, only other thing I noticed is that the HNT amount for the validator list item doesn't convert to local currency when using the toggle. Thats the way it works it is on the consensus list too though.

oh, good catch. I'll fix that.

**Comment by tyler-whitman** [comment]
> > Code looks good and working well for me, only other thing I noticed is that the HNT amount for the validator list item doesn't convert to local currency when using the toggle. Thats the way it works it is on the consensus list too though.
> 
> oh, good catch. I'll fix that.

Looks good to merge when you are ready!

**Pull request merged**